content/people.md

```diff
@@ -2,7 +2,35 @@
 
 # People
 
-- Human 1
-- Human 2
+## Team as cards
+<div class="team-container">    <!-- Team as simple cards -->
+
+  <div class="team-member">
+    <img src="logo.png" alt="Team Member 1" class="team-photo">
```

Comment:
in my local test I don't see these pictures

Reply:
I couldn't find the right path again. I wonder if this is again a baseurl issue?

Reply:
I inserted real images now :)